web/src/lib/i18n/en.json

```diff
@@ -900,7 +899,7 @@
   "removed_api_key": "Removed API Key: {name}",
   "removed_from_archive": "Removed from archive",
   "removed_from_favorites": "Removed from favorites",
-  "removed_from_favorites_count": "Removed {count} from favorites",
+  "removed_from_favorites_count": "{count, plural, other {Removed #}} from favorites",
```

Comment:
Why would this be better?

Reply:
"Removed" can be inflected in other languages.

Reply:
Oh, interesting. TIL